### PR TITLE
Remove global libkb.G

### PR DESCRIPTION
**Pull request by akalin-keybase**
Initialize it in main functions only. Remove test instances and
other unused instances.

Also fix build tag in dokan test file.

### CODE REVIEW

File: libkbfs/test_common.go [hunk 1]
```diff
@@ -79,6 +80,32 @@ func MakeTestBlockServerOrBust(t logger.TestLogBackend,
...
+}
+
+func (t testLogOutput) Info(fmt string, args ...interface{})    { t.log("I", fmt, args) }
+func (t testLogOutput) Error(fmt string, args ...interface{})   { t.log("E", fmt, args) }
+func (t testLogOutput) Debug(fmt string, args ...interface{})   { t.log("D", fmt, args) }
+func (t testLogOutput) Warning(fmt string, args ...interface{}) { t.log("W", fmt, args) }
+func (t testLogOutput) Profile(fmt string, args ...interface{}) { t.log("P", fmt, args) }
+
+func newTestRPCLogFactory(t testLogger) rpc.LogFactory {
+	return rpc.NewSimpleLogFactory(testLogOutput{t}, nil)
```
Comment (akalin-keybase) [inline libkbfs/test_common.go]: This'll cause all RPCs to be logged, but only if tests are run to point to an external server

File: env/context.go [hunk 2]
```diff
@@ -74,9 +74,15 @@ func NewContextFromGlobalContext(g *libkb.GlobalContext) *KBFSContext {
 	return c
 }
 
-// NewContext constructs a context
+// NewContext constructs a context. This should only be called once in
+// main functions.
 func NewContext() *KBFSContext {
-	return NewContextFromGlobalContext(getLibkbG())
```
Comment (jzila) [inline env/context.go]: can you get rid of `getLibkbG` now?

File: libkbfs/keybase_service_util.go [hunk 3]
```diff
@@ -37,21 +35,6 @@ var adminFeatureList = map[keybase1.UID]bool{
 	"b848bce3d54a76e4da323aad2957e819": true, // Surya, "modalduality"
 }
 
-// EnableAdminFeature returns true if admin features should be enabled
-// for the currently-logged-in user.
-func EnableAdminFeature(ctx context.Context, config Config) bool {
```
[the remark this answers is not in the document]
Comment (jzila) [reply libkbfs/keybase_service_util.go]: Please leave this in. We use it often enough and I'd really rather not have to dig through commits to resurrect code.

**Comment by maxtaco** [comment]
if it compiles it's OK with me. there is no more `libkb.G`

**Comment by maxtaco** [comment]
(And it had to work with KBFS for Circle-CI to pass when I merged it in keybase/client)

File: env/context.go (hunk 2 above)
Comment (akalin-keybase) [reply env/context.go]: Oops, meant to do that! Done.

File: libkbfs/keybase_service_util.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (akalin-keybase) [reply libkbfs/keybase_service_util.go]: Done, made it take a `libkb.RunMode` param. Next time it's needed, will need to plumb that down somehow, but shouldn't be too bad.

File: libdokan/createfile_test.go [hunk 4]
```diff
@@ -1,3 +1,9 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+// +build windows
```
Comment (jzila) [inline libdokan/createfile_test.go]: Hah I had `gorename` failing without this too.

**Pull request merged**